docs/cluster-broadcast.md

```diff
@@ -0,0 +1,25 @@
+# Broadcast commands to all Redis Cluster nodes
```

Comment:
Oh, I forgot to write... I am waiting for @uglide's opinion and so I kept the doc format just in case he prefers it over the example format.

Yes, we should definitely keep only of one these two formats.

Reply:
I'll weight. Example only because we can at least compile at test it. There's a docgen plan - no need to support two. Approving.!

Reply:
It'll be compiled, yes, but it won't be tested.

Reply:
That's a first step. Examples aren't currently tested in general.